README.md

```diff
@@ -245,8 +257,9 @@ seed_examples:
 > [!IMPORTANT]
 > Upon release, the taxonomy repository is only accepting contributions from Wikipedia and is capped at 50 contributions. If you want to add knowledge to the taxonomy repository, please fill out this [InstructLab Knowledge Submission Registration](https://docs.google.com/forms/d/1VWJ_XPwH3gBTIXCabpWc0I5pjWIlXETMSFKXc8fpgkA/viewform?edit_requested=true) form and await acceptance! Please do not add contributions if you do not receive the confirmation email. Thank you!
 
-While skills are foundational or performative, knowledge is based more on answering questions that involve facts,
-data, or references.
+While skills are foundational or performative, knowledge is based more on answering questions that involve facts, data, or references.
+
+Knowledge is supported by documents, such as textbook, technical_manual, encyclopedia, journal, or magazine. The document type is the last node in the taxonomy tree, for example: `science/animals/tabby_cat/encyclopedia`. The `qna.yaml` is in this final node.
```

Comment:
```suggestion
Knowledge is supported by documents, such as a textbook, technical manual, encyclopedia, journal, or magazine. The document type is the last node in the taxonomy tree, for example: `science/animals/tabby_cat/encyclopedia`. The `qna.yaml` is in this final node.
```

Also, not sure of the utility of putting the doc type in the path. It is also possible the knowledge documents come from multiple sources: web page, pdf document, wikipedia article.

Reply:
@bjhargrave This was previously discussed and agreed upon that it would be the final node in the path.  I would very much like to see this JUST in the qna.yaml file in the document: section, but the majority of folks said that they'd like to see it in the taxonomy tree for when they just scan/read the tree.

Reply:
I would like to open this up for discussion again :-)